kaldo/interfaces/tdep_io.py

```diff
@@ -7,6 +7,7 @@
 from kaldo.helpers.logger import get_logger
 from kaldo.grid import Grid
 from sparse import COO
+from kaldo.interfaces.common import ForceConstantData, ensure_replicas
```

Comment:
In general, I am not a big fan of helper classes, or tooling classes, unless strictly needed. When possible we should avoid them. 

An alternative approach could be to add these tools to the ForceConstants object (or at least within the ForceConstants module).